### PR TITLE
Update merchants.yml

### PR DESCRIPTION
**Pull request by serhack**
Add my web integrations :D

### CODE REVIEW

File: _data/merchants.yml [hunk 1]
```diff
@@ -78,6 +78,8 @@
       url: https://github.com/MalMen/PHP-Monero
     - name: Monero-PHP (PHP)
       url: https://github.com/PsychicCat/monero-php
+    - name: Monero Payments Library PHP
```
Comment (MaxXor) [inline _data/merchants.yml]: Please put the language `PHP` between `(` and `)` to keep this page consistent.

Comment (serhack) [inline _data/merchants.yml]: Oh thanks for suggestion. I will correct it

**Comment by serhack** [comment]
done @MaxXor

**Comment by cryptochangements34** [comment]
Hey @serhack I think that the WooCommerce plugin might fit a little better under the "Payment Gateways" section.

**Comment by serhack** [comment]
Yup @cryptochangements34 , then moving

**Pull request merged**